.github/workflows/release.yml

```
@@ -32,6 +32,7 @@ jobs:
       - name: Create Release Pull Request
         uses: changesets/action@v1
         with:
+          version: pnpm changeset version && pnpm i
```

Ah my bad 🙈 I think we also need to add `--no-frozen-lockfile` so it can be updated in CI (Astro core has the same config)
```suggestion 
          version: pnpm changeset version && pnpm i --no-frozen-lockfile
```